independent-publisher-connectors/Shopify Admin Store/apiProperties.json

```diff
@@ -0,0 +1,151 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "",
+          "scopes": [
+            "read_analytics",
```

Comment:
@thatapiguy Please make sure that you are documenting all these scores in the readme.md under obtaining credentials section.